src/game.js

```diff
@@ -0,0 +1,60 @@
+function Game() {
+  this.rolls = [];
+  this.score = 0;
+  this.frame = 1;
+  this.frameRollCount = 1;
+}
+
+Game.prototype.roll = function(roll) {
+  if (this.frame >= 10) {
+    this.finalFrame(roll);
+    } else {
+    if (roll === 10){
+      this.rolls.push("X");
+      this.frame++;
+    } else if ((roll + this.rolls[this.rolls.length -1] === 10) && (this.frameRollCount == 2)) {
```

Comment:
Could this code be made clearer by writing helper functions that would describe the intention of conditional statements like `roll + this.rolls[this.rolls.length -1] === 10`?